spec/requests/api_stories/move_workflows/1_police_to_police/1_1_person_with_police_national_computer_spec.rb

```diff
@@ -150,6 +150,9 @@
     police1
     police2
 
+    # Assign police1 location to Serco supplier
+    create :supplier_location, location: police1
```

Comment:
Shouldn't this specify a supplier too, otherwise it will create a new supplier association. Either the comment is wrong or this should pass a supplier parameter I think - the examples in the other workflows look correct.

Reply:
See https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/878/commits/3557320883b4e6f9e534335cb84f434dfb3bfa94